change/@microsoft-fast-foundation-2ada25c3-3fb4-48ba-893d-e64d11f7f095.json

```
@@ -0,0 +1,7 @@
+{
+  "type": "prerelease",
+  "comment": "Fix bug in toolbar click handler when a slotted element has child elements",
+  "packageName": "@microsoft/fast-foundation",
+  "email": "20542556+mollykreis@users.noreply.github.com",
+  "dependentChangeType": "patch"
```

```suggestion
  "dependentChangeType": "prerelease"
```

This will need to be updated to prerlease.